src/index.js

```diff
@@ -10,10 +10,23 @@ const addEventsUrl = baseUrl + '/addEvents';
 const uploadLogsUrl = baseUrl + '/api/uploadLogs';
 
 let logGroupOptions = {};
-if (process.env['LOG_GROUP_OPTIONS']) logGroupOptions = JSON.parse(process.env['LOG_GROUP_OPTIONS']);
-let defaultParserName = process.env['PARSER_NAME'];
+let userLogFile;
+let userParserName;
+let userServerHost;
```

Comment:
These should probably be explicitly initialized to null if that's the intent

Reply:
Sure--I thought leaving undefined would be ok, but yeah--will set to null.